### ISSUE TITLE
Feature request: add comparison to other languages

### ISSUE THREAD

**Issue by magnetophon** [paolomarrone/Zampogna#1]
Thanks for open sourcing this, I find it a very interesting project!

A lot of the features and goals seem similar to [faust](https://faust.grame.fr):
* Minimalistic
* Data flow
* Declarative
* Modular
* suitable for the web

Also the syntax is not too different.

What are some of the advantages and disadvantages of Zampogna as compared to faust?
Might be interesting to add to the website and or a  future README.md on here.

**Comment by sletz** [comment]
Hi @magnetophon, you can find some elements here: https://listserv.cuit.columbia.edu/scripts/wa.exe?A0=MUSIC-DSP and in the SMC https://doi.org/10.5281/zenodo.6573430, comparing in particular how each system handle WDF models. @droosenb who wrote the `wdmodels.lib` will probably refine at some point.

**Comment by magnetophon** [comment]
Thanks, @sletz.
I read in the blog that representing WDFs was the main reason faust was not suitable, but then I also thought: we have WDFs in faust now.
I had tried reading the discussion in the mailinglist, but most of it went over my head.

**Comment by paolomarrone** [comment]
Thanks @magnetophon for your post.
Ciaramella, for sure, shares some objectives with Faust. As far as we have nothing against Faust, we are experimenting new approaches for expressing DSP algorithms more "naturally". For example, the **full** declarative approach (the order of assignments does not matter + SSA) and the synchronous data flow formalism allow for an high level of abstraction and flexibility. The WDF happens to be a good showcase for demonstrating this. The example is both under test/ and in the article that @sletz kindly reported here (Thanks). Not only, it also shows that the creation of feedbacks is "free" (the compiler does not add delays) and this solves a lot of problems. About `wdmodels.lib`, besides some criticisms we highlighted in the article, it is an "ad-hoc" solution... where in Ciaramella the wdf use-cases are treated generally. 
Oh, and I don't find the syntaxes of Faust and Ciaramella similar. :)

Anyways, I added the README.md for some references. Sorry for being so late about that (these are extremely busy days). Thanks!
